packages/@guardian/source-foundations/src/accessibility/accessibility.test.ts

```diff
@@ -0,0 +1,45 @@
+import { transform } from 'lightningcss';
+import { focusHalo, focusHaloSpaced } from './focus-halo';
+import { visuallyHidden } from './visually-hidden';
+
+it('should provide valid CSS for focusHalo', () => {
+	expect(() =>
+		transform({
+			code: Buffer.from(
+				`* {
+					${focusHalo}
+				}`,
+				'utf8',
+			),
+			filename: '',
+		}),
+	).not.toThrowError();
```

Comment:
I would much rather this read like: `expect(isValidCSS(focusHalo)).toBe(true)`.

If you want to be really fancy, you could write [a custom matcher](https://jestjs.io/docs/expect#custom-matchers-api) and have `expect(focusHalo).toBeValidCSS()`.

Reply:
completely agree — a lot clearer and concise